src/amber/controller/helpers/responders.cr

```diff
@@ -60,6 +60,12 @@ module Amber::Controller::Helpers
       end
     end
 
+    def set_response(body, status_code = 200, content_type = Content::TYPE[:html])
```

Comment:
Did this method definition changed in any way? It seem like it did not but yet the method was moved